ntbs-service/deployments/temporarily-unavailable.yml

```diff
@@ -1,21 +1,21 @@
-﻿# Coming-soon page app
+﻿# Temporarily-unavailable page app
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: coming-soon
+  name: temporarily-unavailable
 spec:
   replicas: 1
   selector:
     matchLabels:
-      app: coming-soon
+      app: temporarily-unavailable
   template:
     metadata:
       labels:
-        app: coming-soon
+        app: temporarily-unavailable
     spec:
       containers:
-        - name: coming-soon-app
-          image: "ntbscontainerregistry.azurecr.io/coming-soon"
+        - name: temporarily-unavailable-app
+          image: "ntbscontainerregistry.azurecr.io/temporarily-unavailable"
```

Comment:
I think you need to update the `publishComingSoonPageImage.ps1` script so that it builds an image with this tag. (Apologies if you've done this and I missed it - it's a long review!)

Reply:
I hadn't done, but have now!